digikeyscraper/core.py

```diff
@@ -201,11 +207,10 @@ def download_pdf(csv_dir, pdf_dir):
                 logger.error("    urllib2.URLError for %s" % url)
                 continue
             """
-
+            pbar.update(1)
             time.sleep(0.1)  # Limit our HTTP Requests rate slightly for courtesy.
-    logger.info("Expected %d PDFs." % total)
-    logger.info("Downloaded %d PDFs." % total_count)
-    return "success"  # TODO: make this return something more useful haha
+    logger.debug("Expected %d PDFs." % total)
+    logger.debug("Downloaded %d PDFs." % total_count)
```

Comment:
Suggest to switch to f-string syntax. For example,
```suggestion
    logger.debug(f"Downloaded {total_count} PDFs.")
```